dlog/commitment/src/fft.rs

```diff
@@ -0,0 +1,122 @@
+use ark_ec::ProjectiveCurve;
+use ark_ff::{Field, One, FftField};
+use ark_poly::domain::DomainCoeff;
+use rayon::prelude::*;
+use std::ops::MulAssign;
```

Comment:
btw, it looks like most of this code is taken from https://github.com/arkworks-rs/algebra/blob/86d804bbe6a75f876dddc29dd7973724c84d1a03/poly/src/domain/utils.rs#L78 is there a reason not to use the functions provided by arkworks?

Reply:
I didn't use it because it's private -- I guess we could open a PR over there to expose it publicly if @Pratyush et al are open to it being public.

Reply:
What is the context for using these? You can directly invoke these FFT methods via `EvaluationDomain::fft`

Reply:
Oh cool -- for whatever reason I didn't realize I could just call those. Deleted this file and switched to calling `ifft_in_place` directly.